MonMaraicher/Features/FarmerDetail/FarmerDetailsViewModel.swift

```diff
@@ -10,36 +10,25 @@ import MapKit
 struct FarmerDetailsViewModel {
 
     let title: String
+    let phoneNumber: String?
+    let productions: [Productions]
     let coordinate: CLLocationCoordinate2D
-    let farmerImages: [String]
     let markerSystemImageName: String
     let directionButtonTitle: String
     let address: String
+    let typeAddress: [String]
```

Comment:
`typeAdress` is an array of string? Im a bit lost lol. Something is wrong in the naming